frb_example/pure_dart/rust/src/lib.rs

```diff
@@ -1,5 +1,9 @@
+/// impl_trait: |Debug|fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result
+/// handle_opaque: HideData
+/// add_box: ExoticOptionals
 mod api;
-mod bridge_generated;
+mod bridge_generated; /* AUTO INJECTED BY flutter_rust_bridge. This line may not be accurate, and you can change it according to your needs. */
+mod bridge_generated_bound;
```

Comment:
I thought it was part of api.rs.
If I add bridge_generated, I have to generate bridge_generated twice.
I also implemented ItmImpl(for trait) for Enum. This can be difficult to deal with.

Looking forward to more ideas.

Reply:
Not sure what is the problem... Just generate bridge_generated.rs once